xml/containers-podman-overview.xml

```diff
@@ -64,8 +64,7 @@
   </para>
 
   <para>
-   By default, &podman; requires root privileges. To enable rootless mode for
-   the current user, run the following command:
+   By default, &podman; requires root privileges. Before enabling rootless mode for the current user, check whether there is an entry for the user in the <filename>/etc/subuid</filename> file. If the entry exists, the user can work with &podman; in rootless mode. If there is no entry for the current user in the file, add an entry using the following command:
```

Comment:
There's not really any "default" or way to "enable" rootless mode.

Suggestion: `To run podman without root privileges, subuids/subgids must be assigned to the user running podman. If there is no entry in /etc/..., run ...`

Reply:
Perfect! Thank you very much!